### PR TITLE
Form Translations

### PR DESCRIPTION
**Pull request by alexandriaroberts**
The Form is taking data from translations.

### CODE REVIEW

**Comment by alexandriaroberts** [comment]
Hey@MicheleCynowicz Thank you very much for letting me know!  Yes, I will try to review the changes today and will get back to you!

File: components/Drawer.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
       </v-list-tile>
       <v-list-tile href="http://vuevixens.com/" target="_blank">
         <v-list-tile-title class="subheading">{{ $t("menu.shop") }}</v-list-tile-title>
-      </v-list-tile>
+      </v-list-tile> -->
```
Comment (MicheleCynowicz) [inline components/Drawer.vue]: One tiny fix requested here - I just noticed this extra html-comment closer that doesn't close anything. Can you remove this please?

**Comment by alexandriaroberts** [comment]
> this looks good. Let's :shipit: 👍

Thank you Michele! 🎉

**Comment by MicheleCynowicz** [comment]
closing this PR in favor of the cleaned-up version found here: https://github.com/VueVixens/website/pull/91

**Pull request closed without merging**